lambdas/profiles/read/index.js

```diff
@@ -1,39 +1,27 @@
 const { S3 } = require("aws-sdk");
 const s3 = new S3({
-  apiVersion: "2006-03-01"
+  apiVersion: "2006-03-01",
+  params: { Bucket: process.env.BUCKET_NAME }
 })
 
-module.exports.handler = async function handler ({
-  event,
-  headers,
-  pathParameters
-}) {
-
-  const key = pathParameters.profileName
-  const bucket = process.env.BUCKET_NAME
-
-  const keyExists = await exists(key)
+module.exports.handler = async function handler (event, context) {
 
   console.log("DEBUG:::",
-    "HEADERS:::", headers,
     "EVENT:::", event,
-    "KEY:::", key,
-    "BUCKET:::", bucket,
-    "Key exists:::", keyExists
+    "BUCKET:::", s3,
   )
 
   const params = {
-    Bucket: process.env.BUCKET_NAME,
-    Key: pathParameters.profileName,
+    Key: event.pathParameters.profileName,
   }
 
   const payload = await s3.getObject(params).promise()
```

Comment:
wrap this in a `try {} catch() {}` and return a http error accordingly
and also i would delete the stack
before redeploying with this change